src/app/admin/beneficiaries/beneficiaries-list/beneficiaries-list.component.scss

```diff
@@ -82,3 +82,17 @@ h1 {
     }
   }
 }
+
+mat-select {
+  ::ng-deep {
```

Comment:
Can we use something different then `ng-deep`? It is deprecated.

https://angular.io/guide/deprecations#deep--and-ng-deep-component-style-selectors

Reply:
one that Angular actually recommends, and that’s prepending any ng-deep rule with the :host modifier
Example - styles : [':host ::ng-deep h1 { color:red; }']

Reply:
@valeraf